docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -76,9 +78,19 @@ \section{Critical Assumptions}
 		\centering
 		\caption{Failure Mode and Effect Analysis Table}
 		\begin{adjustbox}{width=\paperwidth}
-			\begin{tabular}{c|c|c|c|c|c|c}
+			\begin{tabular}{|l|l|l|l|l|l|l|}
 				\hline
-				\textbf{Component} & \textbf{Failure Modes} & \textbf{Effects of Failure} & \textbf{Causes of Failure} & \textbf{Recommended Action} & \textbf{SR} & \textbf{Ref.} \\
+				\textbf{Component}              & \textbf{Failure Modes}                                   & \textbf{Effects of Failure}                                                     & \textbf{Causes of Failure}                                                                   & \textbf{Recommended Action}                                                             & \textbf{SR} & \textbf{Ref.} \\
+				\hline
+				General                         & Loss or theft of Personal Identifiable Information (PII) & Legal consequences, loss of user trust                                          & Malicious internal or external actors                                                        & Encryption of PII data at rest and in transit                                           & ~           & ~             \\
```

Comment:
I'm not sure if `loss of user trust` is a Failure effect since that doesn't have to do with the operation of the app.
> Failure effect: The consequence(s) a failure mode has on the operation, function, or status of an item. It can be divided into immediate effect and system effect

Also `Causes of Failure`, `Recommended Action`, and `SR` should be lists. I recommend copying how I setup the table (I have made changes since you last looked at it). https://github.com/arkinmodi/project-sayyara/pull/144

Reply:
Removed loss of user trust and added enumerations